.eslintrc.js

```diff
@@ -25,76 +25,82 @@ module.exports = {
     'plugin:react/recommended',
   ],
 
-  plugins: [
-    'babel',
-    'react',
-    'json',
-    'import',
-  ],
+  plugins: ['babel', 'react', 'json', 'import', 'prettier'],
 
   globals: {
-    '$': 'readonly',
+    $: 'readonly',
     document: 'readonly',
     QUnit: 'readonly',
     window: 'readonly',
   },
 
   rules: {
+    'prettier/prettier': 'error',
+    'space-before-function-paren': [
+      'error',
+      { anonymous: 'always', named: 'never' },
+    ],
+    indent: 'off',
     'arrow-parens': 'error',
     'no-tabs': 'error',
-    'no-mixed-operators': 'error',
+    'no-mixed-operators': 'off',
```

Comment:
https://github.com/prettier/prettier/issues/187 prettier attempts to make output HUMAN readable, check out specifically this:

![image](https://user-images.githubusercontent.com/4448075/81993131-afb5fb80-960a-11ea-90b6-22f412316899.png)

and what it now is formatted to:
![image](https://user-images.githubusercontent.com/4448075/81993161-bd6b8100-960a-11ea-8113-fd755fb82992.png)